NitroxLauncher/Pages/OptionPage.xaml.cs

```diff
@@ -73,5 +77,35 @@ private void OnLogicPropertyChanged(object sender, PropertyChangedEventArgs args
                 OnPropertyChanged(nameof(PathToSubnautica));
             }
         }
+
+        private void ZeroTierInstall(object sender, RoutedEventArgs e)
+        {
+            // check if zerotier is installed
+            if (File.Exists(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86), "ZeroTier", "One", "ZeroTier One.exe")))
+            {
+                MessageBox.Show("Error: ZeroTier is already installed!", "ZeroTier Installer", MessageBoxButton.OK, MessageBoxImage.Warning);
+                return;
+            }
+            // install zerotier
+            DownloadFile("https://download.zerotier.com/RELEASES/1.6.2/dist/ZeroTier%20One.msi", Path.Combine(GetDownloadFolderPath(), "ZeroTier One.msi"));
+            Process RunInstaller = new Process()
+            {
+                StartInfo = new ProcessStartInfo()
+                {
+                    FileName = Path.Combine(GetDownloadFolderPath(), "ZeroTier One.msi")
+                }
+            };
+            RunInstaller.Start();
+            RunInstaller.WaitForExit();
+        }
+        public static void DownloadFile(string remoteFilename, string localFilename)
+        {
+            WebClient client = new WebClient();
+            client.DownloadFile(remoteFilename, localFilename);
```

Comment:
Please dispose the client after use, or make a static client that is used by the entire launcher.

```suggestion
            client.DownloadFile(remoteFilename, localFilename);
            client.Dispose();
```